api_app/serializers.py

```diff
@@ -176,16 +179,20 @@ def filter_connectors(self, serialized_data: Dict) -> List[str]:
         cleaned_connectors_list = []
         selected_connectors = []
 
-        # get values from serializer
         connectors_requested = serialized_data.get("connectors_requested", [])
+        run_all = len(connectors_requested) == 0
```

Comment:
could you add a comment regarding why the logic regardin `run_all` is different between analyzers and connectors? I know this is right but for future developers this would be really hard to understand.

Reply:
it's the same as analyzers. i will make it clear in both `filter_analyzers` and `filter_connectors` functions then